project/src/engines/hoomd/project.py

```diff
@@ -32,34 +32,60 @@ def run_hoomd(job):
     structure = ff.apply(filled_box)
 
     write_gsd(structure, job.fn("init.gsd"), ref_distance=rd, ref_energy=re)
-    # NOTE: create_hoomd3_forcefield depends on PR
-    # https://github.com/mosdef-hub/mbuild/pull/871
+    # ref_distance: angstrom -> nm
+    # ref_energy: kcal/mol -> kJ/mol
+    # ref_mass: dalton -> amu
     snapshot, forcefield, ref_vals = create_hoomd3_forcefield(
-        structure, ref_distance=10, ref_energy=1 / 4.184
+        structure, ref_distance=0.1, ref_energy=1 / 4.184, ref_mass=0.9999938574
```

Comment:
The units in mBuild + foyer are already in nm, and the energies are already in kJ/mol. Do we need to do these conversions here?

Reply:
yes, I think so  because the parmed structure expects [angstrom, kcal, and dalton](https://github.com/mosdef-hub/mbuild/blob/master/mbuild/formats/hoomd_forcefield.py#L68-L72)

Reply:
OK. So you pass the value you divide by to get from the parmed units to the nm/kJ/amu units. So `ref_distance` should be 10 and `ref_energy` should be 1/4.184

Reply:
🤦‍♀️ Yes, you're right. Sorry I realize this is how you had it and I just got confused. I've fixed it and updated the note above to make it more clear to myself!